ex03.exs

```diff
@@ -58,9 +58,33 @@ defmodule Ex03 do
         5 elegant use of language features or libraries
 
   """
-
+  
+  # spawn a chunk-mapping process
+  def spawn_map_process(chunk, func) do
+    Task.async(Enum, :map, [chunk, func])
+  end
+  
+  # wait to receive the mapped chunk
+  def recv_mapped_chunk(pid) do
+    Task.await(pid, :infinity)
+  end
+  
+  # divide the collection into chunks for each process
+  def divide_collection(collection, process_count) do
```

Comment:
Line 74 could also be written somewhat more clearly as

~~~ elixir
chunk_size = collection |> Enum.count |> div(process_count)
~~~

(it's a transformation :)